WordPress/src/main/res/values/strings.xml

```diff
@@ -875,6 +875,13 @@
         <item>@string/app_theme_entry_value_dark</item>
     </string-array>
 
+    <!-- Feature configuration -->
```

Comment:
We might want to consider marking these strings as `translatable=false`. I don't think we need translations for them. Wdyt?